### PR TITLE
adding new resources to cicd script

### PR DESCRIPTION
**Pull request by lukiwlosek**
Adding new resources as mentioned here: https://github.com/aws-ia/cloudformation-github-resource-providers/issues/18

Let me know if I have missed anything and this is the right format, also one thing I was unsure of is the `GITHUB_REPO_ID` value as this needs to be a number not a string, if this has to be as string and fails, this would mean it would have to be changed on the resource side.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/github/cicd.yml [hunk 1]
```diff
@@ -227,6 +253,60 @@ Resources:
...
+        EnvironmentVariables:
+        - Name: RESOURCE_PATH
+          Type: PLAINTEXT
+          Value: "placeholder-for-path-to-resource"
+        - Name: GITHUB_ORG
+          Type: PARAMETER_STORE
+          Value: "cep-github-org"
+        - Name: GITHUB_REPO_ID
+          Type: PARAMETER_STORE
+          Value: 123456789
```
Comment (ericzbeard) [inline release/github/cicd.yml:304]: Change this to "cep-github-repo-id" and I will create a parameter store entry.

**Review by ericzbeard — CHANGES_REQUESTED** [review]

File: release/github/cicd.yml (hunk 1 above)
Comment (ericzbeard) [reply release/github/cicd.yml:304]: I'll need to test and see what happens with the value if it's submitted as a string.

Comment (lukiwlosek) [reply release/github/cicd.yml]: pushed this change

**Review by ericzbeard — APPROVED** [review]

**Comment by ericzbeard** [comment]
> @lukiwlosek Where do I find the numeric github repo id? I'm used to referring to repos by name.

I think I found it: https://stackoverflow.com/questions/13902593/how-does-one-find-out-ones-own-repo-id

**Pull request merged**